caiman_napari/core.py

```diff
@@ -1,6 +1,6 @@
 import os
-from . import algorithms
-from . import _mcorr
+from .algorithms import *
+from .algorithms import mcorr
```

Comment:
@ArjunPutcha This line isn't necessary since the previous line imports all of them